Engine/source/platform/platformNet.cpp

```diff
@@ -127,7 +128,35 @@ static const char* strerror_wsa( S32 code )
       E( WSAEACCES );
 #undef E
       default:
-         return "Unknown";
+		  LPWSTR errString = NULL;  // will be allocated and filled by FormatMessage
+
+		  S32 size = FormatMessageW(  FORMAT_MESSAGE_ALLOCATE_BUFFER |
```

Comment:
It actually takes a LPWSTR and that's why I used one, but not sure if passing that to the getReturnBuffer call will cause issues or not. I would probably need some more input on what the best thing to do would be.